.github/workflows/release.yml

```
@@ -25,26 +25,24 @@ jobs:
         with:
           fetch-depth: 0
       # run before Set node , otherwise show Unable to locate executable file: pnpm error
-      - name: Install pnpm
-        uses: pnpm/action-setup@v4
+      - name: Use just
```

actionlint

step must run script with "run" section or run action with "uses" section

(syntax-check)